02-documentation/developer-guides/building-a-nextjs-blog-with-squidex.md

```diff
@@ -0,0 +1,588 @@
+}
+
+.blog-item-title {
+  font-size: 2em;
+  text-decoration: none;
+  color: black;
+}
+
+.blog-title {
+  font-size: 3em;
+}
+```
+
+Now that you have modified the website's stylesheet, you will go on to modify the custom Next.js `App` component of your blog to wrap the application in a header and footer with the `Layout` component you created. In Next.js, a custom app component is used to add a persistent layout to app pages and to add global CSS to your app. You can learn more about it from the [Custom App page of the Next.js documentation](https://nextjs.org/docs/advanced-features/custom-app).
+
+Modify `_app.js` in `squidex-blog/pages` to have the the following content:
+
+```jsx
+import Layout from "../components/layout";
+import "../styles/globals.css";
+
+export default function MyApp({ Component, pageProps }) {
+  return (
+    <Layout>
+      <Component {...pageProps} />
+    </Layout>
+  );
+}
+```
+
+Visit `localhost:3000` if you haven't already. The development server will rebuild your blog. You will see the newly added header and footer.
+
+Now that you have added the styles and a header and footer to your application, you will proceed to set up authentication for requests to the Squidex API.
+
+## Step 5: Preparing to fetch data from the Squidex API
+
+To be able to fetch data from your Squidex App, you need to have a bearer token. A token gives you the permission to create, read or modify content on a Squidex application. But before you can get a token, you need to create a client on Squidex.
+
+Open the dashboard of your blog application(`squidex-blog` or whatever name you chose) from https://cloud.squidex.io. On the left sidebar, click the _cog icon_ at the bottom left corner of the page to open settings. Under the _Security_ section click _Clients_. At the top of the page, you will see a section where you can create a client. In the text box provided under _Add a new client_, type `nextjs-blog` as the name of the client and click _Add Client_ to create the client. Now when you scroll down the page you will notice a new `nextjs-blog` client has been created. By default, this new client comes with the Editor role. In Squidex, roles define which schemas a client has access to and which operations such as reading and writing a client is authorized to do. Since you wouldn't be making any changes to your content from your blog's frontend, you will change the role of the client to Reader i.e. with only read permissions. In the dropdown for _Role_, select _Reader_.
+
+Check the _Allow anonymous access_ checkbox to allow this client to be query published blog posts without an acess token.
+
+Now you have created a new client and allowed anonymous access to published blog posts, you will create a file containing environment variables for your Next.js app. Create a `.env.local` file in the `squidex-blog` folder and add the following contents
+
+```
+SQUIDEX_API_URL=https://cloud.squidex.io/api/content/squidex-blog/graphql
```

Comment:
You should indicate that "<squidex-blog>" is just a placeholder, e.g. with

SQUIDEX_API_URL=https://cloud.squidex.io/api/content/<YOUR_APP>/graphql